hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmMultipartKeyInfo.java

```diff
@@ -35,6 +35,7 @@
   private final ReplicationType replicationType;
   private final ReplicationFactor replicationFactor;
   private TreeMap<Integer, PartKeyInfo> partKeyInfoList;
+  private long parentID;
```

Comment:
Can we add some comments for parentID? Something like what's the meaning of this and when this  parentID will make sense.

Reply:
Sure, I have pushed the changes in new commit. I updated same comments in OmKeyInfo class as well.